android_sample/build.gradle

```diff
@@ -0,0 +1,25 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.4'
+
+        // NOTE: Do not place your application dependencies here; they belong
+        // in the individual module build.gradle files
+    }
+}
+
+allprojects {
+    repositories {
+        google()
```

Comment:
What do these repositories mean?

Reply:
Does it still build if we remove them?

Reply:
As far as I checked, it works after removing these lines for `allprojects { ... }`. I removed them.